license-generator/generate

```diff
@@ -13,11 +13,14 @@ if ! which cabal-plan >/dev/null; then
     echo "$0: the program 'cabal-plan' is required." >&2
     echo "$0:   see Hackage: https://hackage.haskell.org/package/cabal-plan" >&2
     echo "$0:   repo: https://github.com/haskell-hvr/cabal-plan" >&2
```

Comment:
:information_source: This would right now not work because of an issue in `cabal-plan`, but will be fixed upstream as soon as https://github.com/haskell-hvr/cabal-plan/issues/87 is merged. If you want to run `cabal-plan` before this is merged, checkout the fork/branch instead.